packages/integration/api-report.md

```diff
@@ -329,6 +343,9 @@ export interface IntegrationsByType {
   gitlab: ScmIntegrationsGroup<GitLabIntegration>;
 }
 
+// @public
+export function parseGerritJsonResponse(response: Response): Promise<any>;
```

Comment:
I'd prefer `unknown` over `any` for this one! It's a bit more harsh and proper, and then the caller has to cast to `any` if they truly don't want to cast to some real type. Makes it more visible.